src/worker/generate-metadata.ts

```diff
@@ -0,0 +1,18 @@
+interface Metadata {
+  license: string;
+  languages: string[];
+}
```

Comment:
Could you actually write an interface based on the docs here? https://help.keyman.com/developer/cloud/model_info/1.0/

Here's a formal schema: https://api.keyman.com/schemas/model_info.source.json?_ga=2.49804197.285811810.1600962912-50863687.1550158777

```suggestion
/**
 * See: https://help.keyman.com/developer/cloud/model_info/1.0/
 */
interface ModelInfo {
  license: string;
  languages: string[];
}
```

It seems that only license and languages are required properties; all other properties are optional!